R/pairwise-comparisons.R

```diff
@@ -0,0 +1,244 @@
+#' @title Do pairwise Comparisons of Scores
+#'
+#' @description
+#'
+#' Make pairwise comparisons between models. The code for the pairwise
+#' comparisons is inspired by an implementation by Johannes Bracher.
```

Comment:
link to it and give citation if there is one.

Reply:
There isn't - at least not of Johannes' code

Reply:
No problem then.